build/bin.NOTICE

```diff
@@ -158,3 +158,32 @@ Scala XML (lib/org.scala-lang.modules.scala-xml_<VERSION>.jar)
   scala-xml includes software developed at
   LAMP/EPFL (https://lamp.epfl.ch/) and
   Lightbend, Inc. (https://www.lightbend.com/).
+
+ip4s (lib/com.comcast:ip4s-core.jar-<VERSION>.jar)
+  IP Addresses for Scala and Scala.js
+  Copyright 2018 Comcast Cable Communications Management, LLC
+
+Gson (lib/com.google.code.gson:gson.jar-<VERSION>.jar)
+  Copyright 2008 Google Inc.
+
+decline (lib/com.monovore:decline-<VERSION>.jar, lib/com.monovore:decline-effect-<VERSION>.jar)

```

Comment:
If a dependency does not have a NOTICE file, then you don't need to mention it, so this can be removed.